pkgs/development/php-packages/ssh2/default.nix

```diff
@@ -0,0 +1,21 @@
+{ buildPecl, lib, pkgs }:
```

Comment:
I'm wondering if it wouldn't possible to pass `libssh2` instead of passing `pkgs`.

Reply:
indeed, that seems to work